docs/content/common-subs/required-packages.md

```diff
@@ -116,6 +250,140 @@
         ``` title="kind - https://kind.sigs.k8s.io/docs/user/quick-start/"
         curl.exe -Lo kind-windows-amd64.exe https://kind.sigs.k8s.io/dl/v0.14.0/kind-windows-amd64
         ```
+    === "WSL with Ubuntu"  
```

Comment:
Is this a repeat of the previous lines?

Reply:
@clubanderson @KPRoche I just realized that I needed to split the installation instructions to conform with the "General" tip and "Quickstart" tip distinctions, so I had to mve the instructions for docker and kind down to the "quickstart" tip.